### ISSUE TITLE
Update bundled PhoneGap to 1.4.0

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#62]
```
PhoneGap 1.4.0 is due out the week of January 30, 2012

```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 28 Jan 2012 at 12:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 28 Jan 2012 at 12:29
- Added labels: **Milestone-Release1.2.7**

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 2 Feb 2012 at 12:00
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
- Changed state: **Fixed**
